osbs/cli/main.py

```diff
@@ -206,15 +223,7 @@ def cmd_get_build(args, osbs):
             DOCKERFILE
 
             {dockerfile}
-
-            BUILD LOGS
-
-            {logs}
-
-            PACKAGES
-
-            {packages}
-
+            {logs}{packages}
```

Comment:
I'm not sure why but it seems each empty line is rendered with ```\n```, see [sample output](https://gist.github.com/vrutkovs/dffd745242230137e0e3346b12a08259).